chalice/awsclient.py

```diff
@@ -179,6 +186,19 @@ def _is_settling_error(self, error):
             return True
         return False
 
+    def invoke_function(self, name, context=None, payload=None):
```

Comment:
What's the `context` param used for?

Reply:
Missed removing it. The first POC had the ability to pass a custom context as well.